rmgpy/molecule/molecule.py

```diff
@@ -701,7 +701,22 @@ def applyAction(self, action):
                     raise gr.ActionError('Unable to update Bond due to CHANGE_BOND action: Invalid order "{0}".'.format(action[2]))
         else:
             raise gr.ActionError('Unable to update GroupBond: Invalid action {0}.'.format(action))
-        
+
+    def get_bond_string(self):
+        """
+        Represent the bond object as a string (eg. 'C#N'). The returned string is independent of the atom ordering, with
+        the atom labels in alphabetical order (i.e. 'C-H' is possible but not 'H-C')
+        :return: str
+        """
+        bond_symbol_mapping = {0: '~', 1: '-', 1.5: ':', 2: '=', 3: '#'}
```

Comment:
I asked @mjohnson541 what his preference was (since he is leading the effort for adding hydrogen bonds), and he liked `~` the best. I can change this though if need be.